cime_config/usermods_dirs/NEON/WREF/shell_commands

```diff
@@ -1,3 +1,7 @@
 ./xmlchange NEONSITE=WREF
 ./xmlchange PTS_LON=238.04162
 ./xmlchange PTS_LAT=45.81637
+./xmlchange RUN_STARTDATE=2019-01-01
+./xmlchange DATM_YR_ALIGN=2019,DATM_YR_START=2019
+./xmlchange STOP_N=39
```

Comment:
@wwieder to add the ability to have STOP act differently for control than transient

```
if   [ "`./xmlquery CLM_NML_USE_CASE --value`" =~ "*_transient"  ]; then
   ./xmlchange STOP_N=39
fi
```

We could add an "else" to have it do something different for control cases.

Reply:
We'll also add a shebang line of

`#!/bin/bash`

to the top of the script

Reply:
This is complete in the PR for you to review @ekluzek

Reply:
OK, we have this working now as we want.